src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/mapView.js

```diff
@@ -68,29 +74,131 @@ class MapViewVisualization extends Visualization {
         }
 
         let defaultMapStyle = 'mapbox://styles/mapbox/light-v9';
+        let accentColor     = [1,234,146];
         if (document.getElementById("root").classList.contains("dark")){
             defaultMapStyle = 'mapbox://styles/mapbox/dark-v9';
+            accentColor     = [222,162,47];
         }
 
+        let preparedDataPoints = []
+        let computed           = this.prepareDataPoints(parsedData,preparedDataPoints,accentColor);
+
+        const scatterplotLayer = new deck.ScatterplotLayer({
+            data: preparedDataPoints,
+            getFillColor: d => d.color,
+            getRadius: d => d.radius
+        })
+        //
+        let latitudeMatch  = parsedData.latitude !== undefined && parsedData.latitude !== null;
+        let latitude       = latitudeMatch ? parsedData.latitude : computed.latitude;
+        let longitudeMatch = parsedData.longitude !== undefined && parsedData.longitude !== null;
+        let longitude      = longitudeMatch ? parsedData.longitude : computed.longitude;
+        // TODO : Compute zoom somehow.
```

Comment:
this should be done in this PR. What is the status / problem here?

Reply:
As agreeed we'll merge it without computing zoom.